build-tools/scripts/RunTests.targets

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="RunAllTests" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <_TopDir>$(MSBuildThisFileDirectory)..\..</_TopDir>
+  </PropertyGroup>
+  <Import Project="$(_TopDir)\Configuration.props" />
+  <UsingTask AssemblyFile="$(MSBuildThisFileDirectory)..\..\bin\Build$(Configuration)\xa-prep-tasks.dll" TaskName="Xamarin.Android.BuildTools.PrepTasks.SetEnvironmentVariable" />
+  <PropertyGroup>
+    <XAIntegratedTests Condition=" '$(HostOS)' == 'Windows' ">False</XAIntegratedTests>
+    <_Runtime Condition=" '$(RUNTIME)' != '' ">$(RUNTIME)</_Runtime>
+    <_Runtime Condition=" '$(RUNTIME)' == '' And '$(HostOS)' != 'Windows' ">mono</_Runtime>
+    <_NUnit>$(_Runtime) packages\NUnit.ConsoleRunner.3.7.0\tools\nunit3-console.exe</_NUnit>
+    <_Test Condition=" '$(TEST)' != '' ">--test=&quot;$(TEST)&quot;</_Test>
+    <_XABuild Condition=" '$(HostOS)' == 'Windows' ">$(_TopDir)\bin\$(Configuration)\bin\xabuild.exe</_XABuild>
```

Comment:
Couldn't/shouldn't we *always* just invoke `$(_TopDir)\bin\$(Configuration)\bin\xabuild`, and let the `xabuild` script sort it out?